azure-pipeline.release.yml

```diff
@@ -1,73 +1,53 @@
-trigger: none
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
```

Comment:
Does this mean that when I push a tag to one of the allowed branches the release pipeline will get triggered?

Reply:
I do not believe that tags are branch specific. When you push a tag, that will automatically run the "publish" phase.